modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -3129,10 +3129,21 @@ Verify that this is the intended constraint and that it is spelled correctly.
 m|Neo.ClientNotification.Statement.SubqueryVariableShadowing
 |Title
 a|Variable in subquery is shadowing a variable with the same name from the outer scope.
-|SeverityLevel
-m|INFORMATION
+|Description
+|Variable in subquery is shadowing a variable with the same name from the outer scope.
+If you want to use that variable instead, it must be imported into the subquery using importing WITH clause. (the shadowing variable is: v)
```

Comment:
or it has to be ($msg) instead of (%s)? Because `%s` just means string value of any type

Reply:
This is the old description, and they all are marked with just `(%s)`.

Reply:
I don't know if they always show the same message. Sometimes (%s) = (the shadowing variable is: v), other times (Relationship `r` was repeated), and so on. I guess it means that it is a message, but I don't see such a parameter in the code as for the new GQL codes.

Reply:
probably, we're looking in a wrong place. Old descriptions are done differently from the new GQL statuses. I found this PR https://github.com/neo-technology/neo4j/pull/20016/commits/b86461bcebc690329e6bcfaa0232f0ba6eb33490. Maybe it can be helpful. Let's use `%s`, but I think it's a little weird to see it in the docs

Reply:
I think we can ask Linnea to review your PR

Reply:
Someone should review it for sure.